packages/mobile/src/send/FeeEducation.tsx

```diff
@@ -22,7 +22,7 @@ class FeeEducation extends React.PureComponent<Props> {
           <Text style={[fontStyles.telephoneHeadline, styles.text]}>{t('feeEducation')}</Text>
         </View>
         <View style={styles.buttonContainer}>
-          <CircleButton onPress={navigateBack} solid={true} disabled={false} />
+          <CircleButton onPress={() => navigateBack(true)} solid={true} disabled={false} />
```

Comment:
This will fail the lint checks, we don't allow lambda definitions in JSX.
In the future, you can run `yarn test:all` to see if your changes will pass CI gates.